### PR TITLE
eng-common sync should not run on hosted pools

### PR DESCRIPTION
**Pull request by scbedd**

### CODE REVIEW

**Review by mikeharder — APPROVED** [review]

File: eng/pipelines/eng-common-sync.yml [hunk 1]
```diff
@@ -36,7 +36,8 @@ pr:
       - eng/common
 
 pool:
-  vmImage: windows-2019
+  name: azsdk-pool-mms-win-2019-general
```
Comment (benbp) [inline eng/pipelines/eng-common-sync.yml:39]: Can we try the ubuntu pool? Queueing is a lot faster for it.

**Pull request merged**